src/main/java/uk/gov/justice/digital/delius/service/AttendanceService.java

```diff
@@ -35,7 +35,7 @@ public AttendanceService(final ContactRepository contactRepository) {
     static boolean forEntityBoolean(final String booleanStr) {
         if (booleanStr == null)
             return false;
-        return booleanStr.trim().equals("1");
+        return booleanStr.trim().equals("Y");
```

Comment:
There is TypesTransformer::ynToBoolean  if you prefer

Reply:
Will use that. Thanks.